packages/graphql-server/src/resolvers/office/offices.d.ts

```diff
@@ -0,0 +1,275 @@
+/**
+ * Request body used to set the address of a new office
+ */
+export interface CreateOfficeAddressModel {
```

Comment:
These types are in TS Defintions

Reply:
@willmcvay so we discussed and maintain type definitions in the repo to avoid schema changes in the future